src/index.html

```diff
@@ -9,7 +9,7 @@
   <body>
     <h1>Calendar</h1>
     <main class="calendar">
-      <div class="calendar__container">
+      <div class="calendar__container calendar__container--start-day--wed calendar__container--month-length--30">
```

Comment:
I can't do this because linter crashes :
      <div class="calendar__container 
      calendar__container--start-day--wed 
      calendar__container--month-length--30">